### PR TITLE
Various clean up & fixes

### PR DESCRIPTION
**Pull request by metral**
* Cleanup test utils
* Add Pulumi test debugging
* Bump aws-iam-authentictator version to 1.13.7
* Switch to Gi notation for resources used in k8s test Deployment

### CODE REVIEW

File: nodejs/eks/examples/tests/migrate-nodegroups/nginx-ing-cntlr.ts [hunk 1]
```diff
@@ -66,8 +66,8 @@ export function create(
     // Assemble the resources.
     // Per: https://itnext.io/kubernetes-ingress-controllers-how-to-choose-the-right-one-part-1-41d3554978d2
     const resources: input.core.v1.ResourceRequirements = {
-        requests: {memory: "1024Mi"},
-        limits: {memory: "2048Mi"},
+        requests: {memory: "1Gi"},
```
Comment (lblackstone) [inline nodejs/eks/examples/tests/migrate-nodegroups/nginx-ing-cntlr.ts]: Why is this change needed?

Comment (metral) [reply nodejs/eks/examples/tests/migrate-nodegroups/nginx-ing-cntlr.ts]: See:
- https://github.com/pulumi/pulumi/issues/2967
- Slack [thread](https://pulumi.slack.com/archives/C8R654Y6S/p1563819398010100) from today
